examples/bin/start-druid-main.py

```diff
@@ -50,6 +50,7 @@
 MIDDLE_MANAGER = "middleManager"
 TASKS = "tasks"
 INDEXER = "indexer"
+ZK = "zk"
```

Comment:
I think it might be better to just use the full name `zookeeper` since we are not doing an abbreviation of the other services either.